### PR TITLE
claimfreecoins.io

### PR DESCRIPTION
**Pull request by gork7777**
issue: #79723

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -261,6 +261,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+claimfreecoins.io##div[id^="-adspace"]
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:264]: It doesn't apply on my end. Try `claimfreecoins.io##div[id$="-adspace"]`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (gork7777) [reply EnglishFilter/sections/specific.txt]: Changed.

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/specific.txt]: Ok, thanks.

Comment (gork7777) [reply EnglishFilter/sections/specific.txt]: You're welcome.

**Pull request merged**